client/src/components/overview/ImageNavigator.jsx

```diff
@@ -2,24 +2,72 @@ import React from 'react';
 import './imageNavigator.css';
 
 const ImageNavigator = (props) => {
+  const handleClick = e => {
+    let imageId = e?.target?.dataset?.imageId;
```

Comment:
Do you want this to fall through to the `props` handler whether or not there is an image id?